ee_tests/src/quickstart_analytic.spec.ts

```diff
@@ -98,38 +99,49 @@ describe('Creating new quickstart in OSIO', () => {
-      await expect(spaceDashboardPage.stackReportDependencyCardTotalCount.getText())
-        .toContain(quickstart.dependencyCount.total);
-      await expect(spaceDashboardPage.stackReportDependencyCardAnalyzedCount.getText())
-        .toContain(quickstart.dependencyCount.analyzed);
-      await expect(spaceDashboardPage.stackReportDependencyCardUnknownCount.getText())
-        .toContain(quickstart.dependencyCount.unknown);
-      support.writeScreenshot('target/screenshots/analytic_report_success_' + spaceName + '.png');
-    } catch (e) {
-      support.writeScreenshot('target/screenshots/analytic_report_fail_' + spaceName + '.png');
-      throw e;
-    }
-
-    await spaceDashboardPage.analyticsCloseButton.clickWhenReady();
+    await dashboardPage.ready();
+
+    let codeBasesCard = await spaceDashboardPage.getCodeBaseCard();
+    expect(await codeBasesCard.getCount()).toBe(1, 'number of codebases on page');
+
+    let githubName = browser.params.github.username;
+    let codeBases = await codeBasesCard.getCodeBases();
+    expect(codeBases.length).toBe(1, 'number of codebases');
+    expect(codeBases[0]).toBe('https://github.com/' + githubName + '/' + spaceName);
+
+    let workItemsCard = await spaceDashboardPage.getWorkItemsCard();
+    expect(await workItemsCard.getCount()).toBe(0, 'number of workitems on page');
+
+    let pipelinesCard = await spaceDashboardPage.getPipelinesCard();
+    expect(await pipelinesCard.getCount()).toBe(1, 'number of pipelines on page');
+
+    let pipelines = await pipelinesCard.getPipelines();
+    expect(pipelines.length).toBe(1, 'number of pipelines');
+    expect(pipelines[0].getApplication()).toBe(spaceName, 'application name on pipeline');
+    expect(pipelines[0].getStatus()).toBe(BuildStatus.COMPLETE, 'build status');
+    expect(pipelines[0].getBuildNumber()).toBe(1, 'build number');
+
+    let deploymentsCard = await spaceDashboardPage.getDeploymentsCard();
+    expect(await deploymentsCard.getCount()).toBe(1, 'number of deployments on page');
+
+    let applications = await deploymentsCard.getApplications();
+    expect(applications.length).toBe(1, 'number of applications');
+    expect(await applications[0].getName()).toBe(spaceName, 'deployed application name');
+    expect(await applications[0].getStageVersion()).toBe('1.0.1', 'deployed application stage version');
+    expect(await applications[0].getRunVersion()).toBe('1.0.1', 'deployed application run version');
+
+    let analyticsCard = await spaceDashboardPage.getAnalyticsCard();
+    let totalCount = await analyticsCard.getTotalDependenciesCount();
+    let analyzedCount = await analyticsCard.getAnalyzedDependenciesCount();
+    let unknownCount = await analyticsCard.getUnknownDependenciesCount();
+
+    expect(totalCount).toBeGreaterThanOrEqual(0, 'total dependencies count');
+    expect(analyzedCount).toBeGreaterThanOrEqual(0, 'total analyzed count');
+    expect(unknownCount).toBeGreaterThanOrEqual(0, 'total unknown count');
```

Comment:
Shouldn't it be 0? Are there valid use cases when it can be non-zero?

Reply:
This seems correct - there might be an edge case where the value is set to zero. Is this the edge case that you are trying to handle?